### DISCUSSION TITLE
web login failed with status: -100

### DISCUSSION THREAD

**Discussion by leechangseok917** [storycraft/node-kakao#379] [Q&A]
web login failed with status: -100

위 현상은 어떤경우에 일어나는건가요?ㅠㅠ

**Comment by ghost** [comment]
인증되지 않은 디바이스가 로그인에 시도했을때 반환되는 status Code입니다.

**Reply by leechangseok917** [reply]
그러면 첫인증은 어떻게따야되나요?

**Reply by raravel** [reply]
example 폴더에 device-registration.ts 참고하세요.

https://github.com/storycraft/node-kakao/blob/1058808a75270efd3021ecb75acd2cdc81b42abf/examples/device-registration.ts#L1-L62
